scripts/west_commands/runners/openocd.py

```diff
@@ -41,9 +41,13 @@ def __init__(self, cfg, pre_init=None, pre_load=None,
         else:
             search_args = []
 
+        common = path.split(path.split(cfg.board_dir)[0])
+        search_args += ['-s', path.join(common[0], 'common')]
```

Comment:
Is it possible to do this by using `-c` on the openocd command line, like we do the other commands, instead of using a file? If not, I think it might be worth adding `boards/common` to the `config: openocd_search:` path in build/zephyr/runners.yaml. You can do that in cmake/flash/CMakeLists.txt.

The goal is to make this work for out of tree boards, too.

Reply:
I actually think the `openocd_search` approach might be the best overall. Curious what others think.

Reply:
Hi @mbolivar-nordic ,

> Is it possible to do this by using -c on the openocd command line, like we do the other commands, instead of using a file?

I gave some tries without success. I'll ask on the openocd forum.

Reply:
>  I'll ask on the openocd forum.

I think the `openocd_search` approach is probably better, so no worries here!